packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -57,13 +63,18 @@ export default function VisualEditor() {
 	useClipboardHandler( ref );
 	useTypingObserver( ref );
 	useCanvasClickRedirect( ref );
+	const editorStylesRef = useEditorStyles( styles );
+	const mergedRefs = useCallback( mergeRefs( [ ref, editorStylesRef ] ), [
+		ref,
+		editorStylesRef,
+	] );
```

Comment:
When does `styles` change? How can I trigger a change?

@youknowriad We discussed this a few times that adding dependencies here will give problems when passing new functions as a callback ref.

Reply:
Ah I see your comment now https://github.com/WordPress/gutenberg/pull/28233#discussion_r565089599
I'll rebase #27768 in a bit and see what is missing there.